acceptance-tests/src/test/scala/ch/renku/acceptancetests/pages/LoginPage.scala

```diff
@@ -110,7 +110,9 @@ object ProviderLoginPage
   }
 
   def logInButton(implicit webDriver: WebDriver): WebElement = eventually {
-    find(cssSelector(".btn.btn-primary.btn-block.btn-lg")) getOrElse fail("Log In button not found")
+    find(cssSelector(".btn.btn-primary.btn-block.btn-lg"))
```

Comment:
I think the better solution is the one here:

https://github.com/SwissDataScienceCenter/renku/pull/2177/commits/9905efaf7387c0016a2de09b59646d863b098fb6

Using the id will work for both versions of the page (Renku styled and un-styled), so it is unnecessary to support both versions.

Reply:
@ciyer I wasn't aware of the change. I guess you can maybe replace our change with your version when merging.